package.json

```diff
@@ -24,13 +24,15 @@
     "start": "pm2-runtime start pm2.config.js",
     "heroku-postbuild": "echo 'Skipping build'",
     "build": "node scripts/prod.js",
-    "clean": "rm -rf ./packages/client/__generated__ ./queryMap.json ./packages/server/graphql/public/schema.graphql ./packages/server/graphql/private/schema.graphql ./dev ./dist ./build",
+    "clean": "find . \\( -name node_modules -o -name dev -o -name dist -o -name build -o -name '__generated__' -o -name queryMap.json -o -name schema.graphql -o -name resolverTypes.ts \\) -prune | xargs rm -rf",
     "codegen": "node scripts/codegenGraphQL.js",
     "pg:build": "pgtyped -c ./packages/server/postgres/pgtypedConfig.js",
     "pg:migrate": "node-pg-migrate -f ./packages/server/postgres/pgmConfig.js",
     "pg:restore": "node ./scripts/toolbox/pgRestore.js",
+    "db:clean": "docker-compose -f docker/dev.yml down --rmi all --volumes",
```

Comment:
not sure if name `db` here is accurate but I guess it's better to follow the convention